tools/src/commands/GenerateDocsAPIData.ts

```diff
@@ -67,6 +67,15 @@ const PACKAGES_MAPPING: Record<string, CommandAdditionalParams> = {
   'expo-local-authentication': ['LocalAuthentication.ts'],
   'expo-localization': ['Localization.ts'],
   'expo-location': ['index.ts'],
+  'expo-maps': [
+    [
+      'index.ts',
+      'apple/AppleMapsView.tsx',
+      'apple/AppleMaps.types.ts',
+      'google/GoogleMapsView.tsx',
+      'google/GoogleMaps.types.ts',
+    ],
+  ],
```

Comment:
i wasn't sure what to add so i added everything

Reply:
Exporting `index` should be enough, since at quick lat at file, it includes all wanted exports:
* https://github.com/expo/expo/blob/main/packages/expo-maps/src/index.ts

If changing to single export root leads to some missing entries, in most cases it's a package export structure issue.

Reply:
i tried that and it only generated docs for a few of the apis for some reason, as i added more files it added more apis, so i just added them all

Reply:
It's fine to leave it as-is from docs perspective, but would be nice to audit package exports separately, to make sure that the current structure is an expected one.